ctre/basemotorcontroller.py

```diff
@@ -190,12 +194,14 @@ def setInverted(self, invert: bool):
         :param invert:
             Invert state to set.
         """
-        self.invert = invert
-        super().setInverted(invert)
+        self.invert = int(invert)
+        super()._setInverted_2(invert)
 
     def getInverted(self) -> bool:
         """:returns: invert setting of motor output"""
-        return self.invert
+        if self.invert in [0, 1]:
```

Comment:
`(0, 1)` would probably be better here - a tuple of constants can go into the constant pool, iirc.